meetings/Minutes_2023-01-08.md

```diff
@@ -0,0 +1,93 @@
+Present: Vicki, Yeshoda, Madeleine, Sophie, Ahmed, Elliot;
```

Comment:
```suggestion
Present: Vicki, Yeshoda, Madeleine, Sophie, Ahmed, Elliott;
```

Reply:
sorry for spelling your name wrong so many times :(